extensions/wikia/Captcha/scripts/Captcha.js

```diff
@@ -0,0 +1,65 @@
+$(function () {
+	'use strict';
+
+	/**
+	 * JS for handling captchas We default to using reCaptcha, however if we encounter any problems, will
+	 * fall back to using Fancy Captcha instead.
+	 */
+	var Captcha = {
+		/**
+		 * WikiaMobile, Wikia One, and some automated tests do not use captcha
+		 */
+		useCaptcha: !window.wgUserSignupDisableCaptcha,
```

Comment:
This global is actually specific to the user signup page, and it only really gets set from the user signup controller. I think it shouldn't matter in any user-facing way, b/c if you're not on the Special:UseSignup page, this global var won't be set, but maybe the comment should reflect the fact that it's specific to user signup, just so people don't get confused in the future.